duckduckgo/duckduckgo-autofill
# Position tooltip above input when no space below

**Reviewer:** @GioSensation 
**Asana:** https://app.asana.com/0/0/1204207719532788/f

## Description
Display tooltip above input whenever there is limited space below

![incontext-signup-tooltip-position](https://user-images.githubusercontent.com/635903/226754547-0f2be7c5-5ad0-4527-a29f-8691fc8a10c2.gif)

## Steps to test
1. Build into chrome extension build
2. Go to https://www.axios.com/newsletters
3. Click on email input at bottom of the screen
4. Confirm tooltip opens above input

<img width="2160" alt="Screenshot 2023-03-21 at 22 20 39" src="https://user-images.githubusercontent.com/635903/226754933-9e8134bf-152c-44bc-ac1b-d395318a3129.png">
